libs/login/build.gradle

```diff
@@ -14,6 +14,12 @@ android {
         compileSdk rootProject.compileSdkVersion
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
+
+        // Optional: used to autofill username and password fields at login on debug build only
+        buildConfigField "String", "DEBUG_WPCOM_LOGIN_EMAIL", "\"\""
+        buildConfigField "String", "DEBUG_WPCOM_LOGIN_USERNAME", "\"\""
+        buildConfigField "String", "DEBUG_WPCOM_LOGIN_PASSWORD", "\"\""
+        buildConfigField "String", "DEBUG_WPCOM_WEBSITE_URL", "\"\""
```

Comment:
After thinking about this again, actually this might be useful to keep the behaviour of having these properties easy to apply on local machines without risk of commiting. See: 0d1dedb.

And demo that this works fine:

https://github.com/user-attachments/assets/fb77d899-e9c8-47ab-84e7-2925a0f9db6f

Reply:
> Btw, why not doing the same with fluxc too? 🤔

Sure, addressed in f366ab563c0a72d376fd340c143ed73bdef35f3e

Reply:
Awesome, thanks for that @wzieba , everything LGTM! 🚀